C/c4Document.cc

```diff
@@ -472,6 +491,90 @@ bool c4doc_save(C4Document *doc,
     return false;
 }
 
+bool c4doc_setExpiration(C4Database *db, C4Slice docId, uint64_t timestamp, C4Error *outError)
+{
+    C4Document *doc = c4doc_get(db, docId, true, outError);
```

Comment:
To check if the doc exists, it's cheaper just to call `db->get(docID, kMetaOnly).exists()`.